react/features/calendar-sync/functions.any.js

```diff
@@ -160,6 +184,7 @@ function _getURLFromEvent(event, knownDomains) {
         'gi');
     const schemeRegExp
         = new RegExp(`${APP_LINK_SCHEME}${linkTerminatorPattern}+`, 'gi');
+    const excludePattern = new RegExp('/static/');
```

Comment:
Looks fine to me. Here are some questions:

1. I'm not familiar with the js regex object. Is it necessary to instantiate an new instance every time here and in _checkPattern?
2. Not serious musing: I'm wondering if a different check is needed here in the case someone is hosting the jitsi-meet index page on a url that includes /static/...but unlikely to be worth the hassle.

Reply:
Thanks for the review! For point 1: you're totally right, I forgot to remove the double instantiation. Fixed, thanks!
About point 2: Aaron said that the /static/ path is a hardcoded setting in nginx, so I'm safe using it. We could have a setting for it though, not sure if that's worth the additional hassle.

Reply:
Did you push the changes to fix the double instantiation?

Reply:
Damn, I forgot that line... Thanks, should be okay now